fluent-plugin-google-cloud.gemspec

```diff
@@ -19,6 +19,7 @@ eos
   gem.test_files    = gem.files.grep(/^(test)/)
   gem.require_paths = ['lib']
 
+  gem.add_runtime_dependency 'chronic_duration', '~> 0.10.6'
```

Comment:
This is a third party lib that parses strings to duration: https://github.com/henrypoydar/chronic_duration

Is it a good practice to use something like this? Maybe we can fork it if necessary. Or simply use a regex instead?